Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/RedundantTypeAnnotationConfiguration.swift

```diff
@@ -0,0 +1,11 @@
+import SwiftLintCore
+
+@AutoApply
+struct RedundantTypeAnnotationConfiguration: SeverityBasedRuleConfiguration {
+    typealias Parent = RedundantTypeAnnotationRule
+
+    @ConfigurationElement(key: "severity")
+    var severityConfiguration = SeverityConfiguration<Parent>(.warning)
+    @ConfigurationElement(key: "ignored_annotations")
+    var ignoredAnnotations = Set<String>(["IBInspectable"])
```

Comment:
The word `ignore` might be more appropriate than `ignored` because the configuration element is "configuring the rule to _ignore_ certain annotations". Also, the code base [appears](https://github.com/search?q=repo%3Arealm%2FSwiftLint%20ignore&type=code) to use `ignore`/`ignores` more than `ignored`.

Side note, the PR title and description mention ignore "attributes", not "annotations". I'm just pointing out for consistency purposes.

Great work!

```suggestion
    @ConfigurationElement(key: "ignore_annotations")
    var ignoreAnnotations = Set<String>(["IBInspectable"])
```

Reply:
You're right it should be attributes and not annotations, it took me a while to figure out which was the actually correct wording for Swift 😅 I'll fix that up!

As for the use of past tense, for me `ignored_attributes` suggests that the given attributes will be ignored by the linter while `ignore_attributes` suggests that this property is a flag that enables ignoring all attributes. Though I agree that for the sake of concistency it is better here to use `ignore` 👍

Reply:
> As for the use of past tense, for me `ignored_attributes` suggests that the given attributes will be ignored by the linter while `ignore_attributes` suggests that this property is a flag that enables ignoring all attributes.

I'm with @tonell-m about this. `ignore_attributes` sounds like a boolean flag.

> Though I agree that for the sake of concistency it is better here to use `ignore` 👍

Can you provide an example where it is really used like that? I can only find flags following this name pattern. `ignored_attributes` seems like a better fit, to be honest.